.github/workflows/ci.yml

```
@@ -0,0 +1,60 @@
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: "21"
+          distribution: "temurin"
+          cache: gradle
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Build with Gradle
+        run: ./gradlew build -x test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)